highway/crud_registry.go

```diff
@@ -3,203 +3,98 @@ package highway
 import (
 	context "context"
 	"fmt"
-	"log"
 	"net/http"
 
-	"github.com/duo-labs/webauthn/protocol"
-	"github.com/duo-labs/webauthn/webauthn"
-	"github.com/gorilla/mux"
-	"github.com/patrickmn/go-cache"
+	"github.com/gin-gonic/gin"
 	rtv1 "github.com/sonr-io/blockchain/x/registry/types"
-	rt "go.buf.build/sonr-io/grpc-gateway/sonr-io/blockchain/registry"
+	rt "go.buf.build/grpc/go/sonr-io/blockchain/registry"
 )
 
-// StartRegisterName starts the registration process for webauthn on http
-func (s *HighwayServer) StartRegisterName(w http.ResponseWriter, r *http.Request) {
-	// get username/friendly name
-	vars := mux.Vars(r)
-	username, ok := vars["username"]
-	if !ok {
-		JsonResponse(w, fmt.Errorf("must supply a valid username i.e. foo.snr"), http.StatusBadRequest)
-		return
+// RegisterNameStart starts the registration process for webauthn on http
+func (s *HighwayServer) StartRegisterName(c *gin.Context) {
```

Comment:
Is there no rpc variant of these requests?

Reply:
Only for RegisterName - We can make this cohesive after we create monorepo.